demos/notebooks/demo_pipeline_cnmfE.ipynb

```diff
@@ -69,16 +82,85 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "fnames = ['data_endoscope.tif']  # filename to be processed\n",
-    "fnames = [download_demo(fnames[0])]"
+    "# set up logging\n",
+    "logging.basicConfig(format=\"{asctime} - {levelname} - [{filename} {funcName}() {lineno}] - pid {process} - {message}\",\n",
+    "                    filename=None, \n",
+    "                    level=logging.WARNING, style=\"{\") #logging level can be DEBUG, INFO, WARNING, ERROR, CRITICAL\n",
+    "\n",
+    "# set env variables \n",
+    "os.environ[\"MKL_NUM_THREADS\"] = \"1\"\n",
```

Comment:
I think we do this in the environment entry scripts? Not sure we need to repeat that here

Reply:
It can't hurt, esp for people who are in dev mode.